ts/components/messages/MessageAgenda.tsx

```diff
@@ -266,15 +266,95 @@ const FakeItemComponent = (
 class MessageAgenda extends React.PureComponent<Props, State> {
   // Ref to section list
   private sectionListRef = React.createRef<any>();
+  private idTimeoutProgress?: number;
 
   constructor(props: Props) {
     super(props);
     this.state = {
-      itemLayouts: []
+      itemLayouts: [],
+      firstSectionListUpdate: true,
+      elementVisible: 0,
+      isLoadingProgress: false,
+      isContinuosScrollEnabled: true
     };
     this.loadMoreData = this.loadMoreData.bind(this);
   }
 
+  public componentWillUnmount() {
+    if (this.idTimeoutProgress !== undefined) {
+      clearTimeout(this.idTimeoutProgress);
+    }
+  }
+
+  // checks if the last section is shown, if yes disables continuos scroll
+  private checkIfIsLastSection(): boolean {
+    if (this.props.lastDeadlineId.isNone()) {
+      // No are deadlines
+      this.setState({
+        isContinuosScrollEnabled: false
+      });
+      return false;
+    } else {
+      // tslint:disable-next-line: no-let
+      for (const section of this.props.sections) {
```

Comment:
Instead of 2 nested for you could use a more functional approach
see https://github.com/teamdigitale/io-app/pull/1441#discussion_r358156937